maquetado/setup/scss/titles.scss

```diff
@@ -0,0 +1,18 @@
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+
+.title {
+  font-size: $text-big;
+  margin: 50px 55px;
+  position: relative;
+
+  &::before {
+    bottom: -25px;
```

Comment:
If you want to avoid using negative values, you can use top and right instead: 

``` 
  top: calc(100% - 25px);
  right: calc(100% - 50px);
```

Also, you could use top and right without `calc()`